src/Controls/src/Core/Platform/Windows/Extensions/FormattedStringExtensions.cs

```diff
@@ -127,5 +128,54 @@ public static void UpdateInlines(this TextBlock textBlock, Label label)
 
 			return Tuple.Create(run, span.TextColor, span.BackgroundColor);
 		}
+
+		internal static void RecalculateSpanPositions(this TextBlock control, Label element, IList<double> inlineHeights)
```

Comment:
if we make the other public we should do it for this as well

Reply:
Changed.